lib/scripts/seed_test_db.ts

```diff
@@ -55,8 +55,8 @@ async function seedUsers(users: UserJSON[]): Promise<Prisma.BatchPayload> {
 
     const newUser: User = {
       id: user.id,
-      firstName: user.first_name,
-      lastName: user.last_name,
+      firstName: user.first_name || 'test',
+      lastName: user.last_name || 'test',
```

Comment:
see this PR/ branch https://github.com/pharmaboxfydp/pharmabox/blob/f84daa8ba4f2688e7218994ad7f7e7847be0b249/lib/scripts/seed_test_db.ts#L58 that would overwrite this implementation. the aforementioned implementation is prefered